tests/conftest.py

```diff
@@ -285,6 +285,12 @@ def es(app):
         current_search_client.indices.delete_template('*')
 
 
+@pytest.yield_fixture()
```

Comment:
Minor thing here: `pytest.yield_fixture` is deprecated in favor of `pytest.fixture`. This should be propagated to the rest of the fixtures as well.